.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
:+1: But then, maybe we should rename the default folder, right? On the other hand, we don't want to conflict with travis cache some day.